src/auth/awsclient.ts

```diff
@@ -58,6 +58,15 @@ interface AwsSecurityCredentials {
   Expiration: string;
 }
 
+/**
+ * Interface defining the AWS security-credentials retrieved from environment variables.
+ */
+interface AwsSecurityCredentialsFromEnvironment {
+  accessKeyId: string;
+  secretAccessKey: string;
+  token?: string;
+}
```

Comment:
Instead of duplicating interfaces, use the imported `AwsSecurityCredentials` interface